evaluate_summaries.py

```diff
@@ -29,6 +29,10 @@ def load_model_results_for_subset(data_subset: str, beam_suffix: str = ""):
         }
     else:
         results = {}
+        if "bart" in data_subset:
```

Comment:
this is kinda hacky, but can be simplified when the full-results for pegasus are computed and merged

Reply:
I believe we only had N=100 results for BART as well. I needed to work off of your branch to avoid needing to re-run GEF on BART